apps/web/app/product-detail/components/product-info-tab.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { TabContainer, Tab } from '@lococo/design-system';
+
+interface ProductInfoTabProps {
+  productDetail: string;
+  ingredients: string;
+}
+
+export default function ProductInfoTab({
+  productDetail,
+  ingredients,
+}: ProductInfoTabProps) {
+  const [activeTab, setActiveTab] = useState<'productDetail' | 'ingredients'>(
+    'productDetail'
+  );
+
+  return (
+    <div className="flex w-full flex-col">
+      <TabContainer className="w-full">
+        <Tab
+          label="製品の特徴"
+          variant={'secondary'}
+          className="w-full"
+          onClick={() => setActiveTab('productDetail')}
+          active={activeTab === 'productDetail'}
+        ></Tab>
+        <Tab
+          label="原材料・成分"
+          variant={'secondary'}
+          className="w-full"
+          onClick={() => setActiveTab('ingredients')}
+          active={activeTab === 'ingredients'}
+        ></Tab>
+      </TabContainer>
+      {activeTab === 'productDetail' && (
+        <div className="text-jp-body1 border-b-[0.1rem] border-gray-300 py-[4rem] text-gray-700">
+          {productDetail}
+        </div>
+      )}
+      {activeTab === 'ingredients' && (
+        <div className="text-jp-body1 border-b-[0.1rem] border-gray-300 py-[4rem] text-gray-700">
+          {ingredients}
+        </div>
+      )}
```

Comment:
요거 탭이면 둘 중 하나만 나타나게 될 것 같은데 && 연산자를 통해서 렌더링하는 것보다는 상수로 묶어서 렌더링할 로직을 처리하는 것도 좋을 것 같아용

```suggestion
const tabContentRender = {
  productDetail: (
    <div className="text-jp-body1 border-b-[0.1rem] border-gray-300 py-[4rem] text-gray-700">
      {productDetail}
    </div>
  ),
  ingredients: (
    <div className="text-jp-body1 border-b-[0.1rem] border-gray-300 py-[4rem] text-gray-700">
      {ingredients}
    </div>
  ),
}[activeTab];
```

Reply:
이렇게 분리할 정도로 복잡하지 않다고 생각했어요! 근데 왜이리 가독성이 안좋을까 생각해보니 같은 디자인의 div를 반복적으로 썼네요 아래와 같이 리팩토링하겠습니다!
52db1bd
```
      <div className="text-jp-body1 border-b-[0.1rem] border-gray-300 py-[4rem] text-gray-700">
        {activeTab === 'productDetail' ? productDetail : ingredients}
      </div>
```